src/ugeneui/src/main_window/MainWindowImpl.cpp

```diff
@@ -192,8 +184,8 @@ MainWindowImpl::MainWindowImpl() {
     visitWebAction = NULL;
     viewOnlineDocumentation = NULL;
     checkUpdateAction = NULL;
+    createDesktopShortcutAction = nullptr;
     aboutAction = NULL;
-    openManualAction = NULL;
```

Comment:
Remove the class member from the ".h" file

Reply:
Done